examples/sequential_retrieval.py

```diff
@@ -0,0 +1,422 @@
+"""
+# Recommending movies: retrieval using a sequential model
+
+In this example, we are going to build a sequential retrieval model. Sequential
+recommendation is a popular model that looks at a sequence of items that users
+have interacted with previously and then predicts the next item. Here, the order
+of the items within each sequence matters. So, we are going to use a recurrent
+neural network to model the sequential relationship. For more details,
+please refer to the [GRU4Rec](https://arxiv.org/abs/1511.06939) paper.
+
+Let's begin by importing all the necessary libraries, and setting the
+random seed for reproducibility.
+"""
+
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
```

Comment:
Any reason to do that? At some point I was thinking we can write tests to run these automatically with each backend. But I never got around to doing that. But for it to work, the backend has to not be set within the file.

Did you test with TF and Torch too?

Reply:
Yep, working for all backends!